crates/fuzz/src/fuzz_stats.rs

```diff
@@ -93,10 +111,60 @@ impl FuzzingStatistics {
                     successful: 1,
                     failed: 0,
                     failed_check: 1,
+                    cu_used_max: 0,
+                    cu_used_min: u64::MAX,
```

Comment:
This could be misleading if every transaction fails, as it would show that the minimum amount was u64::MAX. I understand why you approached it this way, but I think we need to consider if there’s a better solution.